pom.xml

```diff
@@ -188,12 +188,6 @@ flexible messaging model and an intuitive client API.</description>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpclient</artifactId>
         <version>4.5.5</version>
```

Comment:
as this dependency already part of parent-dep-management so, should we remove version as well?

Reply:
This section is from the top-level dep-management

Reply:
I left the exclusion on the `pulsar-client/pom.xml` so that it get only applied there

Reply:
I misread it, I thought this change was made in pulsar-client-pom.xml.